.github/workflows/test_and_deploy.yml

```diff
@@ -47,7 +47,7 @@ jobs:
     if: github.event_name == 'push' && github.ref_type == 'tag'
     runs-on: ubuntu-latest
     steps:
-    - uses: neuroinformatics-unit/actions/build_sdist_wheels@main
+    - uses: neuroinformatics-unit/actions/build_sdist_wheels@v1
```

Comment:
All fine, but these (unrelated) changes could come in a  micro-PR to clean up the diff.

Reply:
Yep, exactly. I'd
```
git checkout -b update-actions main
git cherry-pick <flibble>
git push --set-upstream origin update-actions
```
... which I think is what you're saying.

Just so that unrelated changes are factored out: When I [removed python 3.8](https://github.com/SainsburyWellcomeCentre/WAZP/pull/32#issuecomment-1431264780) it was marginal: I probably _should_ have done that in a separate PR.

To be clear: all of this is suuuper picky ... but more if we want to converge onto the zen of perfect dev workflow.

[Here's one opinion about this that I found](https://gavinr.com/clean-pull-request-diffs/).

Reply:
I agree with that opinion and it's good that you brought it up (even if you consider it picky). It's good to build good habits in practice. I will merge #42 ahead of this one, for that clean diff